cnms/service/networkmonitor.go

```diff
@@ -178,6 +178,4 @@ func main() {
 		time.Sleep(time.Duration(timeout) * time.Second)
 		nm = nil
 	}
-
-	log.Close()
```

Comment:
Dont we have to defer log.close() this ?

Reply:
since the above `for` loop in has no `break` this is unreachable code and we don't currently ever calling `Close()`. 
so i don't know if we need to, but if it hasn't been a problem i would guess no.

Reply:
its good practice to have defer log.Close()..if someone changes this behaviour tomorrow, then they may fail to add log.close()

Reply:
@tamilmani1989 I read through the log package and I don't think it's safe to call `log.Close()` (even deferred) the way that it's written right now. We could accidentally close stdout or stderr and prevent the go runtime from writing the crash logs during a panic. That needs some work that out of scope for this linting fix